protocol/communities/manager.go

```diff
@@ -1256,25 +1272,39 @@ func (m *Manager) DeleteCategory(request *requests.DeleteCommunityCategory) (*Co
 	return community, changes, nil
 }
 
+// TODO: re-implement me
 func (m *Manager) HandleCommunityDescriptionMessage(signer *ecdsa.PublicKey, description *protobuf.CommunityDescription, payload []byte) (*CommunityResponse, error) {
+	// queue descriptions from signers (one description per signer per community, higher clock wins)
+	// async trigger control node resolution (no more than once per N secs)
+	// when control node is resolved, handle description from control node, if any, then clear the queue
+	// signal community changes
```

Comment:
@cammellos WDYT?

Reply:
As per our offline discussion, resolving the control node for each `CommunityDescription` appears to be excessive. Instead, the contract should be polled periodically, say, once every 5 minutes. If a change of control node is detected, the messages should be re-fetched. This will manage the scenario where a `CommunityDescription` from the new signer is received before we conduct our check.